lib/truffle/rbconfig.rb

```diff
@@ -112,12 +112,21 @@ module RbConfig
   ruby_home = Truffle::Boot.ruby_home
 
   if ruby_home
-    prefix = ruby_home
-    bindir = "#{prefix}/bin"
+    prefix        = ruby_home
+    bindir        = "#{prefix}/bin"
+    extra_bindirs = if Truffle.graalvm?
+                      graalvm_home = Truffle::System.get_java_property 'org.graalvm.home'
+                      [File.expand_path(File.join(graalvm_home, 'bin')),
+                       File.expand_path(File.join(graalvm_home, 'jre', 'bin'))]
```

Comment:
This can be written like:
```ruby
File.expand_path('bin', graalvm_home)
File.expand_path('jre/bin', graalvm_home)
```
which I find much easier to read.

Reply:
good catch